airtime_mvc/application/models/airtime/CcFiles.php

```diff
@@ -314,20 +315,19 @@ private static function validateFileArray(&$fileArray)
         return true;
     }
 
-
     public function getCueLength()
-	{
-		$cuein = $this->getDbCuein();
-		$cueout = $this->getDbCueout();
-		
-		$cueinSec = Application_Common_DateHelper::calculateLengthInSeconds($cuein);
-		$cueoutSec = Application_Common_DateHelper::calculateLengthInSeconds($cueout);
-		$lengthSec = bcsub($cueoutSec, $cueinSec, 6);
-		
-		$length = Application_Common_DateHelper::secondsToPlaylistTime($lengthSec);
-		
-		return $length;
-	}
+	  {
```

Comment:
This indentation seems to have increased the wrong way - should be less

Reply:
Oops. Didn't notice. I have to be more careful next time.